pkg/backend/httpstate/client/client.go

```diff
@@ -97,6 +97,18 @@ func getStackPath(stack StackIdentifier, components ...string) string {
 	return path.Join(append([]string{prefix}, components...)...)
 }
 
+// listPolicyGroupsPath returns the API path to for the given organization with the given
+// components joined with path separators and appended to the organization root.
```

Comment:
Nit: I'm guessing the comments for these two new functions are copied from `publishPolicyPackPath` below, but they're not very specific to each function.

For `listPolicyGroupsPath`, I'd expect a comment more like:

```go
// listPolicyGroupsPath returns the path for an API call to the Pulumi service to list the policy groups
// in a Pulumi organization.
func listPolicyGroupsPath(orgName string) string {
```

Reply:
yeah it seems like all the functions here had a super generic comment, I will make it more specific